examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -382,7 +382,13 @@ static void createMqttTask(void) {
 }
 
 static void initMqtt(void) {
-  if (mqttClient.begin(configuration.getMqttBrokerUri())) {
+  String mqttUri = configuration.getMqttBrokerUri();
+  if (mqttUri.isEmpty()) {
+    Serial.println("MQTT is not configured, skip initialize mqtt client");
```

Comment:
propose to rephrase to

.., skipping initialization of mqtt client

and decide if we want to use capitalize or not. Maybe 

"MQTT URL  is not configured, , skipping initialization of MQTT client